src/Roave/SecurityAdvisories/Version.php

```diff
@@ -46,6 +46,7 @@ private function __construct(array $matches)
 
         $this->flag = Flag::build($matches['flag'] ?? '');
 
+        $numbers = null;
         if (isset($matches['stability_numbers'])) {
             $numbers = self::removeTrailingZeroes(...array_map('intval', explode('.', $matches['stability_numbers'])));
```

Comment:
Instead of `$numbers = null`, we should:

1. `$this->stabilityNumbers` default to `[]`
2. `if (! isset($matches['stability_numbers']) { return; }`

Reply:
Fixed by assigning $this->stabilityNumbers inside the if() after assigning to [] by default